larray/core/array.py

```diff
@@ -5637,6 +5637,15 @@ def to_hdf(self, filepath, key, *args, **kwargs):
         >>> a = ndtest((2, 3))
         >>> a.to_hdf('test.h5', 'a')  # doctest: +SKIP
         """
+        if isinstance(key, Group):
```

Comment:
I would rather use `sheet_name = str(_to_tick(sheet_name))` instead of all this code